src/services/token.js

```diff
@@ -0,0 +1,21 @@
+import { decode } from 'jsonwebtoken'
+
+const setToken = (token) => localStorage.setItem('token', token)
+const getToken = () => localStorage.getItem('token')
```

Comment:
According to [Authentication](https://uranus.docs.apiary.io/#/introduction/authentication?mc=reference%2Fflight%2Fairports%2Fget-iata-list%2F200), user basic information would not be stored in JWT token.